code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -120,3 +120,27 @@
 	)
 	armor = list(melee = 40, bullet = 5, laser = 20,energy = 5, bomb = 35, bio = 100, rad = 50)
 	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+
+//Head of Security
+/obj/item/clothing/head/helmet/space/void/hos
+	name = "heavy security voidsuit helmet"
+	desc = "A special helmet designed for work in a hazardous, low pressure environment. Has an additional layer of armor, and gold trim."
+	icon_state = "rig0-hos"
+	item_state_slots = list(
+		slot_l_hand_str = "sec_helm",
+		slot_r_hand_str = "sec_helm"
+		)
+	armor = list(melee = 60, bullet = 10, laser = 30, energy = 5, bomb = 45, bio = 100, rad = 10)
```

Comment:
Since it is HOS suit it is okay to bump armor for it, because currently it is exactly like security.
I would say `armor = list(melee = 65, bullet = 15, laser = 40, energy = 10, bomb = 50, bio = 100, rad = 15)`

Reply:
I explicitly do not want to do this. The HoS is not intended to be front lining, they are a manager, not a fighter- the new voidsuit is for convenience and easy identification, and because honestly the sprites look rad as hell, but not to buff their combat capabilities. While it might be reasonable to do I'm concerned with the meta implications.

Reply:
I explicitly do not want to do this. The HoS is not intended to be front lining, they are a manager, not a fighter- the new voidsuit is for convenience and easy identification, and because honestly the sprites look rad as hell, but not to buff their combat capabilities. While it might be reasonable to do I'm concerned with the meta implications.

Reply:
Captain is not front liner either, yet their suit has armor as good as mercs do.

Reply:
I also concur with doc here.
A HoS is more likely to lead the fight if they know that they have more robust armor.